### DISCUSSION TITLE
Add align text on Beatmap Attribute Text

### DISCUSSION THREAD

**Discussion by Davpyu** [ppy/osu#23668] [Q&A] [answered]
### Type

Cosmetic

### Bug description

It seems the beatmap attribute text only able to do align center. Its being a problem when i put it as Song Title/Artist and its too long or even just put it so close to corner that it hides/overlap fullscreen (See Image). I try to add more margin but the text itself got align center instead align left like i want.

### Screenshots or videos

![osu_2023-05-26_11-23-53](https://github.com/ppy/osu/assets/42276243/e30a6245-1e4a-4bbc-ab92-e42f55785e13)
here i want to align 2 element of beatmap attribute text to align left on the left corner of my screen
![osu_2023-05-26_11-24-08](https://github.com/ppy/osu/assets/42276243/ad5b85f5-99d3-4365-b07f-1e3324df3247)
it works on this song
![osu_2023-05-26_11-24-22](https://github.com/ppy/osu/assets/42276243/3f0ac092-6066-4a63-817e-816f03e41d99)
i tried on the other song it hides the text

### Version

2023-513.0-lazer

### Logs

-

**Comment by nagi-desuuu** [comment, marked as answer]
But anchors exist...
You can also do that to `Origin`, forcing an element to align, say, to the left.

![image](https://github.com/ppy/osu/assets/75596775/6ab049e9-5cc7-4c09-9fe4-f92f5c230780)
